install-nccl-tests.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+NCCL_TESTS_VERSION="v2.0.0"
+cd $HOME
+git clone -b "$NCCL_TESTS_VERSION" https://github.com/NVIDIA/nccl-tests
+pushd nccl-tests
+# TODO: We need to apply the patch in commit https://github.com/NVIDIA/nccl-tests/commit/0f173234bb2837327d806e9e4de9af3dda9a7043
```

Comment:
Did this still not make its way into a release? That was a year or so back.

Reply:
nccl-tests's latest release (v2.0.0 https://github.com/NVIDIA/nccl-tests/commits/v2.0.0, released Mar 16, 2019) does not take this commit in.

Reply:
Alright then.